NapDatabaseExport/MSSQLServerProvider.cs

```diff
@@ -39,11 +49,18 @@ private string GenerateConnectionString ()
             if (!string.IsNullOrEmpty (Server))
                 connectionString.DataSource = Server;
 
-            if (!string.IsNullOrEmpty (User))
+            if (!string.IsNullOrEmpty(User))
+            {
```

Comment:
Fix the formatting of the braces position and the spacing

Reply:
Ooops, sorry! I'm using default Visual Studio settings. Why is this not compatible with this repo?

Reply:
This is the code formatting we are using in this project